src/components/services/OctoPi.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="item.subtitle && printerstatus == null">
+          {{ item.subtitle }}
+        </template>
+        <template v-else-if="api" && printerstatus !="null">
+          Status: {{ printerstatus }}
+        </template>
+        <template v-if="printerstatus == 'Printing'"
+          >|| Progress: {{ printerprogress }}% <br />
+          Elapsed: {{ printtime }} || Left: {{ printtimeleft }}
+        </template>

```

Comment:
I'd recommend a more concise setup to avoid multiple lines

```suggestion
        <template v-if="item.subtitle && printerstatus == null">
          {{ item.subtitle }}
        </template>
        <template v-else-if="api" && printerstatus !="null">
          {{ printerstatus }}<template v-if="printerstatus == 'Printing'">: {{ printerprogress }}% ({{ printtime }}/{{  printtimeleft }} left)</template>
        </template>

```

If the status is not printing, it'll simply show that status. If printing it should show something like:

Printing: 46% (1:20 / 1:35 left)